### PR TITLE
Fix problem with read_pubkey_file error handling.

### PR DESCRIPTION
**Pull request by BobWall23**
Read_pubkey_file was returning 0 even if it couldn't read the key file.
Fixed to return proper error and success values.
Also removed some crusty test / experimental code from iron-gpg.c

### CODE REVIEW

**Comment by BobWall23** [comment]
Fixes Issue #15

File: iron-gpg.c [hunk 1]
```diff
@@ -3999,14 +3999,25 @@ static int
 read_pubkey_file(const char * login, Key * rsa_key, unsigned char * rsa_fp, unsigned char * cv25519_key,
 				 size_t * cv25519_key_len, unsigned char * cv25519_fp, char * uid)
 {
-	int retval = 0;
+	int retval = -1;
+
+	if (rsa_key	!= NULL) {
```
Comment (coltfred) [inline iron-gpg.c]: Why the funky amount of space in here?

Comment (BobWall23) [inline iron-gpg.c]: F'ing tabs vs. spaces.

**Comment by zmre** [comment]
Again, have seen this same diff in two other PRs. Am using an iPad app right now so I guess it could be having problems?

**Pull request merged**